webapp/src/components/modals/channel_settings/channel_settings_internal.jsx

```diff
@@ -113,21 +115,54 @@ export default class ChannelSettingsModalInner extends PureComponent {
     };
 
     handleProjectChange = (id, value) => {
-        const projectKey = value;
+        let projects = value;
```

Comment:
maybe refactor into ensureArray (reuse for both handle fns)? or maybe not worth it. up to you.

Reply:
I'd rather not introduce a library for this PR, I think that's what you mean. However, it does sound like a nice way to remove that ugly type checking code :stuck_out_tongue_winking_eye: